beta1/integrated-build.json

```diff
@@ -93,8 +93,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```

Comment:
Again - we can't remove anything. This is part of what pops out of "openshift ex generate" in the beta code.

So if you are saying:

"It will not break if you have https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin in the strategy and it will not break if the image is openshift/ruby-20-centos"

Then we are OK. However, this PR still changes the beta-1 instructions, which is a no-go. So you'll need to fix the PR if that's the case.